### PR TITLE
meta-isar/conf: introduce mtda-bullseye distro

### PR DESCRIPTION
**Pull request by chombourger**
Having per-machine DISTRO_APT_SOURCES or DISTRO_APT_PREFERENCES
was resulting in differences in the isar-bootstrap-target builds
and would cause problems in multi-config builds. Moreover, the
"abstraction" layer provided by the MTDA_EXTRA_APT variables
introduced in c610fef was pointless. A mtda-bullseye distro is
instead introduced (it extends debian-bullseye) and includes
backports we require for NanoPi variants such as the R1.

Signed-off-by: Cedric Hombourger <cedric.hombourger@siemens.com>

### CODE REVIEW

File: kas/common/base.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+target: isar-image-base
+
+repos:
+  mtda:
+    layers:
+      meta-isar:
+  isar:
+    url: https://github.com/ilbers/isar.git
+    refspec: master
```
Comment (jan-kiszka) [inline kas/common/base.yml]: Not a good idea. Set this in a nightly CI build or so, but it's obviously nothing for common use. No layer used in production does that.

File: kas/common/base.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    layers:
+      meta:
+      meta-isar:
```
Comment (jan-kiszka) [inline kas/common/base.yml:26]: Are you sure that you use meta-isar? It's generally for demo/testing/reference purposes.

Comment (chombourger) [reply kas/common/base.yml:26]: we do - it ships the nanopi-neo machine

Comment (jan-kiszka) [reply kas/common/base.yml:26]: Ah, ok. As long as you can use those two files, this makes sense.

File: kas/common/base.yml (hunk 1 above)
Comment (chombourger) [reply kas/common/base.yml]: I removed the use of `master` in the following commit `kas/common: use older Isar to work-around build failures`
I can squash these two if you prefer (I felt I should refactor first and then use a fixed refspec but I am happy to change it with the refactoring itself)

Comment (jan-kiszka) [reply kas/common/base.yml]: Given that `master` was used before, squashing is not actually needed. I would just rephrase the last commit towards "use a well-defined Isar version".

Comment (chombourger) [reply kas/common/base.yml]: done

**Pull request merged**